src/_pytest/nodes.py

```diff
@@ -188,26 +195,26 @@ def __init__(
         #: The parent collector node.
         self.parent = parent
 
-        #: The pytest config object.
```

Comment:
Looks like Sphinx silently ignores those comments (and hides the attribute as undocumented) if this comment isn't next to the line actually setting the attribute.